### ISSUE TITLE
Follow DHTML Style Guide Working Group for Grid Keyboard Shortcuts

### ISSUE THREAD

**Issue by JLLeitschuh** [angular-ui/ui-grid#3893]
Would it be better if cell nav were to follow the following guidelines?
http://access.aol.com/dhtml-style-guide-working-group/#grid

I think it would fix some of the problems with being unable to use the arrow keys inside of a cell in a grid to manipulate dropdowns and input boxes.

This is an example of a grid that uses this keyboard setup. http://oaa-accessibility.org/example/18/

**Comment by swalters** [comment]
We modeled the keyboard navigation on excel and google spreadsheet.  Enter in spreadsheets go to next cell which isn't listed in this doc.

**Comment by bialikover** [comment]
Are there any standard in grouping and other features? I've found some issues when implementing grouping and keyboard navigation together

**Comment by mportuga** [comment]
Note: Add help wanted label
